Dockerfile

```diff
@@ -13,3 +10,6 @@ WORKDIR /serialplot
 ADD . /serialplot
 WORKDIR ./build_docker
 RUN cmake ../ && make -j appimage
+
+FROM scratch as exporter
```

Comment:
I'm a bit confused with these changes. What I want to understand is, what happens if I want to build this file as a docker image, instead of running `buildx`? Does this file now contain multiple image definitions?

Reply:
Ah, if the usecase for this is also to have an image to perform development within, this change does not make sense.